### PR TITLE
Add a skeleton driver for Alcatel Web Touch One web phone

### PR DESCRIPTION
**Pull request by clawgrip**
New systems marked not working
------------------------------
Alcatel Web Touch One (model 2840, Terra, Spanish) [Arcade Hacker, ClawGrip]

### CODE REVIEW

File: src/mame/skeleton/webtouchone.cpp [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+{
+	MPC8240(config, m_maincpu, XTAL(32'768'000)); // Actually a Motorola XPC823ZT66A
+
+	SCREEN(config, m_screen, SCREEN_TYPE_LCD); // Hitachi SX19V001-ZZA (7,6'' touch, color, 640x480)
+	m_screen->set_refresh_hz(60); // Guess
+	m_screen->set_size(640, 480);
+	m_screen->set_visarea(0, 640-1, 0, 480-1);
+	m_screen->set_screen_update(FUNC(webtouchone_state::screen_update_webtouchone));
+
+	SPEAKER(config, "mono").front_left();
```
Comment (angelosa) [inline src/mame/skeleton/webtouchone.cpp:135]: If there's one speaker only then this should be `front_center()`

File: src/mame/skeleton/webtouchone.cpp [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+   |      |___________|            | Motorola   |          |_________| |
+   |                               | XPC823ZT66A|           __________ |
+   |                    Xtal       |            |          |74HCT541 | |
+   |                 32.768 MHz    |            |          |_________| |
+   |                               |____________|                      |
+   |                                                                   | 
+   |                                          _______                  |
+   | 3BN62121AAAF KAZZA 01                   |ST 324|                  |
+   |___________________________________________________________________|
+
```
Comment (angelosa) [inline src/mame/skeleton/webtouchone.cpp:42]: Legit puzzled about what RAMDAC this uses, MPC823 datasheet mentions having one for the LCD interface. Where's TFT connector?

Comment (clawgrip) [reply src/mame/skeleton/webtouchone.cpp:42]: Cannot remember, I dumped it four years ago, and I only keep photos of the disconnected PCBs.
There is some more hardware info here: https://www.minitel-alcatel.fr/documents/WebPhones/SES%20Hard%202840.pdf

File: src/mame/skeleton/webtouchone.cpp (hunk 1 above)
Comment (clawgrip) [reply src/mame/skeleton/webtouchone.cpp]: Fixed, thanks!

File: src/mame/skeleton/webtouchone.cpp [hunk 3]
```diff
@@ -0,0 +1,158 @@
+// license:BSD-3-Clause
+// copyright-holders:
+
+/*
+  Skeleton driver for Alcatel Web Touch One.
+  More information and technical manuals: https://www.minitel-alcatel.fr/gamme_webphone.html
+  The phone asks for an adminstrative password for performing a factory reset.
+
+  Hardware info for model 2840:
+   -CPU: Motorola Power PC 823
```
Comment (cuavas) [inline src/mame/skeleton/webtouchone.cpp:10]: PowerPC doesn’t have a space.

File: src/mame/skeleton/webtouchone.cpp [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+	ROM_REGION( 0x008000, "keyboard", 0 )
+	ROM_LOAD( "upd78f0034agc.bin",               0x000000, 0x008000, NO_DUMP )
+ROM_END
+
+} // anonymous namespace
+
+
+//    YEAR  NAME       PARENT  COMPAT  MACHINE      INPUT        CLASS              INIT        COMPANY    FULLNAME  FLAGS
+COMP( 1999, wto2840sp, 0,      0,      webtouchone, webtouchone, webtouchone_state, empty_init, "Alcatel", "Alcatel Web Touch One (model 2840, Terra, Spanish)", MACHINE_IS_SKELETON )
```
Comment (cuavas) [inline src/mame/skeleton/webtouchone.cpp:158]: You don’t need to repeat “Alcatel” in the description.  The product name is just “Web Touch One”.

File: src/mame/skeleton/webtouchone.cpp (hunk 3 above)
Comment (clawgrip) [reply src/mame/skeleton/webtouchone.cpp]: Fixed!

File: src/mame/skeleton/webtouchone.cpp (hunk 4 above)
Comment (clawgrip) [reply src/mame/skeleton/webtouchone.cpp]: Fixed!

**Pull request merged**